### PR TITLE
Create a version using Playwright [Exercice 05 - Install Dependencies]

### PR DESCRIPTION
**Pull request by glaucia86**
Once [Playwright](https://playwright.dev/) is an Open-Source tool and is from Microsoft, I think we could change the example using Playwright! I created this branch and ask you to consider the possibility. It could be a way to promote the tool throughout the Node.js course.

Link: [Exercise - Install packages](https://learn.microsoft.com/en-us/training/modules/create-nodejs-project-dependencies/5-exercise-dependency)

image

### CODE REVIEW

File: node-dependencies/5-exercise-dependency-playwright/package.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  "name": "5-exercise-dependency-playwright",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "playwright test"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```
Comment (manekinekko) [inline node-dependencies/5-exercise-dependency-playwright/package.json:11]:
```suggestion
  "license": "MIT",
```

File: node-dependencies/5-exercise-dependency-playwright/address-parser.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+exports.parse = function parseOrder(order) {
```
Comment (manekinekko) [inline node-dependencies/5-exercise-dependency-playwright/address-parser.js:1]: the exported name and function name should be the same (good practice).

Comment (glaucia86) [reply node-dependencies/5-exercise-dependency-playwright/address-parser.js:1]: Good catch, @manekinekko!
@diberry we will need to update this part of the Learn! :)

Thanks a lot for it, Wassim :)

**Comment by diberry** [comment]
LGTM

**Review by diberry — APPROVED** [review]

**Pull request merged**